balancer/pickfirst/pickfirstleaf/pickfirstleaf.go

```diff
@@ -54,9 +54,17 @@ func init() {
 	balancer.Register(pickfirstBuilder{})
 }
 
-// enableHealthListenerKeyType is a unique key type used in resolver attributes
-// to indicate whether the health listener usage is enabled.
-type enableHealthListenerKeyType struct{}
+type (
+	// enableHealthListenerKeyType is a unique key type used in resolver attributes
+	// to indicate whether the health listener usage is enabled.
+	enableHealthListenerKeyType struct{}
+	// managedByPickfirstKeyType is an attribute key type to inform Outlier
+	// Detection that the generic health listener is being used.
+	// TODO(arjan-bal): Remove this when implementing the dualstack design.
```

Comment:
Sorry for the nit pick. Can we have a linked issue with the TODO. Historically, we haven't done a good job of it, and I'm trying to change that. Thanks.